common/survey/categoryItem.js

```diff
@@ -12,31 +12,37 @@ const keys = {
 
 const props = {
   code: 'code',
-  labels: 'labels'
+  descriptions: 'descriptions',
+  extra: 'extra',
+  labels: 'labels',
 }
 
 // ====== CREATE
-const newItem = (levelUuid, parentItem = null, props = {}) => {
-  return {
-    [keys.uuid]: uuidv4(),
-    [keys.levelUuid]: levelUuid,
-    [keys.parentUuid]: SurveyUtils.getUuid(parentItem),
-    [keys.props]: props,
-  }
-}
+const newItem = (levelUuid, parentItemUuid = null, props = {}) => ({
+  [keys.uuid]: uuidv4(),
+  [keys.levelUuid]: levelUuid,
+  [keys.parentUuid]: parentItemUuid,
+  [keys.props]: props,
+})
 
 // ====== READ
 const getCode = SurveyUtils.getProp(props.code, '')
 
 const getLabels = SurveyUtils.getProp(props.labels)
 
-const getLabel = language =>
-  item =>
-    R.pipe(
-      getLabels,
-      R.prop(language),
-      R.defaultTo(getCode(item))
-    )(item)
+const getLabel = language => item =>
```

Comment:
don't we have a generic method ```Object.getLabel```? if not, let's add it?

Reply:
This one defaults to the item code... We can't make it generic.